### DISCUSSION TITLE
Model management

### DISCUSSION THREAD

**Discussion by Richjanz** [EdgeTX/edgetx#5138] [Q&A]
On my Horus x10x express in model management the models are named (Bonanza, Citabria, Nieuport) but on the SD card in the models folder they are numbered, ( Modei1, Model5, Model14. How do I know which model number corresponds to which plane? I want to delete one plane from the SD card but I don't know which one

**Comment by pfeerick** [comment]
Why can't you delete it using the model manager? It should move it to the
deleted folder ;)

Otherwise, opening labels.yml and searching for your model name to find the
associated filename is probably the simplest option.
